File: Iniciante/URI 1015.ml
let x1, y1, x2, y2 = Scanf.scanf "%f %f\n%f %f" (fun a b c d -> a, b, c, d);;
let distancia = sqrt(((x2 -. x1) ** 2.0) +. ((y2 -. y1) ** 2.0));;
Printf.printf "%.4f\n" (distancia);

File: Iniciante/URI 1015.c
#include <stdio.h>
#include <math.h>
 
int main() { 
    float x1, x2, y1, y2, distancia;

    scanf("%f %f %f %f", &x1, &y1, &x2, &y2);
	distancia = sqrt(pow((x2 - x1), 2.0) + pow((y2 - y1), 2.0));
    printf("%.4f\n", distancia);

    return 0;
}